sycl/test/aot/multiple-devices.cpp

```diff
@@ -8,9 +8,18 @@
 
 // REQUIRES: ioc64, ocloc, aoc
 
-// Produce object file, spirv
-// RUN: %clangxx -fsycl -fsycl-targets=spir64_x86_64-unknown-linux-sycldevice,spir64_gen-unknown-linux-sycldevice,spir64_fpga-unknown-linux-sycldevice %s -c -o %t.o
-// RUN: %clangxx -fsycl -fsycl-link-targets=spir64_x86_64-unknown-linux-sycldevice,spir64_gen-unknown-linux-sycldevice,spir64_fpga-unknown-linux-sycldevice %t.o -o %t.spv
+// 1-command compilation case
+// Targeting CPU, GPU, FPGA
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_x86_64-unknown-linux-sycldevice,spir64_gen-unknown-linux-sycldevice,spir64_fpga-unknown-linux-sycldevice -Xsycl-target-backend=spir64_gen-unknown-linux-sycldevice "-device skl" %s -o %t_all.out
```

Comment:
Just note: "-device skl" means this test won't work on older SKUs. Right?

Reply:
I'm not quite sure, TBH - I've never tried running this on older architectures.
Depending on how our AOT tools change/our compiler options expand in foreseeable future, we may get better ways of specifying the GEN arch.

Reply:
Most likely GEN compiler will utilize Gen9 ISA. Can we use `broadwell` instead?